src/compas_view2/objects/__init__.py

```diff
@@ -33,3 +40,6 @@
 Object.register(Sphere, SphereObject)
 Object.register(Torus, TorusObject)
 Object.register(Arrow, ArrowObject)
+
+if is_assembly:
+    Object.register(Block, MeshObject)
```

Comment:
why not just put the try-except here?

Reply:
ok @brgcode, changed in commit 7d9b35db05acf1bf430d285fa14acb7cbf765a07